src/roi/__tests__/ratio.test.ts

```diff
@@ -0,0 +1,21 @@
+test('ratio of ROI', () => {
+  const mask = testUtils.createMask([
+    [0, 1, 1, 1],
+    [0, 1, 0, 1],
+    [0, 1, 1, 1],
+    [0, 0, 0, 0],
+  ]);
+
+  expect(mask.ratio).toBe(4 / 4);
+});
+
+test('ratio of ROI', () => {
+  const mask = testUtils.createMask([
+    [0, 1, 1, 1, 0, 0, 0],
+    [0, 1, 0, 1, 0, 0, 0],
+    [0, 1, 1, 1, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+  ]);
+  expect(mask.ratio).toBe(7 / 5);
```

Comment:
@stropitek didn't you tell me that the goal was to implement the properties on ROI objects? This test seems to imply that it's on a Mask.

Reply:
My bad, I wanted to test the ratio and unintentionally put the mask ratio test into ROI testing folder.